kie-wb-common-react/import-java-classes/src/components/ImportJavaClasses/ImportJavaClassesWizard.tsx

```diff
@@ -17,14 +17,24 @@
 import * as React from "react";
 import { ModalWizard } from "../ModalWizard";
 import { useImportJavaClassesWizardI18n } from "../../i18n";
-import { ImportJavaClassesWizardFirstStepContent } from "./ImportJavaClassesWizardFirstStep";
+import { ImportJavaClassesWizardFirstStep } from "./ImportJavaClassesWizardFirstStep";
 
-export const ImportJavaClassesWizard: React.FunctionComponent = () => {
+export interface ImportJavaClassesWizardProps {
+  /** Button disabled status */
+  buttonDisabledStatus: boolean;
+  /** Button tooltip message */
+  buttonTooltipMessage?: string;
+}
+
+export const ImportJavaClassesWizard: React.FunctionComponent<ImportJavaClassesWizardProps> = ({
```

Comment:
@yesamer Hi, is this expected to be FunctionComponent? I think we do not need to re-render this every time parent changes. What about `PureComponent` does it make sense here?

Reply:
@domhanak Thank you for your suggestion. I can consider it as a way to improve performances, undoubtedly. Consider I'm a react newbie :), so I would consider having a performance review when the whole component is completed. As an alternative, we could prefer the usage of `useMemo` hook. WDTY @vpellegrino and @karreiro ?

Reply:
Usage of `PureComponent` in place of `ClassComponent` has a sense from a performance perspective.
Since, in our case, we are using `FunctionComponent`, you can use the `useMemo` hook (https://reactjs.org/docs/react-api.html#reactmemo) to reach the same result.